cmd/krew/cmd/update.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 // IndexURI points to the upstream index.
-const IndexURI = "https://github.com/GoogleContainerTools/krew-index.git"
+const IndexURI = "https://github.com/lbb/krew-test-index.git"
```

Comment:
should we revert this or keep until we update the index
OR maybe we should add a `IndexBranch` feature that checks out to a particular branch after cloning WDYT?

Reply:
Oh, that slipped in from local dev. I’ll totally revert it :)
After using pure go-git /& multi-repos we should revisit that idea 👍